benchmark/benchmarkTests.go

```diff
@@ -129,8 +133,11 @@ func SociFullRun(
 	}
 	defer cleanupTask()
 	log.G(ctx).WithField("benchmark", "RunTask").WithField("event", "Start").Infof("Start Run Task")
+	runTask1Start := time.Now()
 	cleanupRun, err := sociContainerdProc.RunContainerTaskForReadyLine(ctx, taskDetails, readyLine)
+	runTask1Duration := time.Since(runTask1Start)
```

Comment:
Can we think of better names than "task1" and "task2"? Maybe "lazyTask" and "localTask"? I'm not totally sure that's clear enough, but it's a little better.

Reply:
Sure, would localTask be the task without the FUSE overhead?

Reply:
Updated.